projects/packages/paypal-payments/package.json

```diff
@@ -15,11 +15,48 @@
 	"license": "GPL-2.0-or-later",
 	"author": "Automattic",
 	"scripts": {
-		"build": "echo 'Not implemented.'",
+		"build": "pnpm run clean && webpack --config ./webpack.config.blocks.js",
 		"build-js": "echo 'Not implemented.'",
 		"build-production": "echo 'Not implemented.'",
 		"build-production-js": "echo 'Not implemented.'",
-		"clean": "true"
+		"clean": "rm -rf dist/ .cache/"
 	},
-	"devDependencies": {}
+	"dependencies": {
+		"@automattic/babel-plugin-replace-textdomain": "workspace:*",
+		"@automattic/format-currency": "1.0.1",
+		"@automattic/jetpack-shared-extension-utils": "workspace:*",
+		"@automattic/jetpack-webpack-config": "workspace:*",
+		"@automattic/remove-asset-webpack-plugin": "workspace:*",
+		"@wordpress/base-styles": "5.22.0",
+		"@wordpress/block-editor": "14.17.0",
+		"@wordpress/components": "29.8.0",
+		"@wordpress/compose": "7.22.0",
+		"@wordpress/data": "10.22.0",
+		"@wordpress/dependency-extraction-webpack-plugin": "6.22.0",
+		"@wordpress/element": "6.22.0",
+		"@wordpress/hooks": "4.22.0",
+		"@wordpress/i18n": "5.22.0",
+		"@wordpress/widgets": "4.22.0",
+		"clsx": "2.1.1",
+		"copy-webpack-plugin": "11.0.0",
+		"email-validator": "2.0.4",
+		"gridicons": "3.4.1",
+		"lodash": "4.17.21",
+		"react": "18.3.1",
+		"react-dom": "18.3.1",
+		"webpack": "5.94.0",
+		"webpack-cli": "6.0.1"
+	},
+	"devDependencies": {
+		"@automattic/jetpack-base-styles": "workspace:*",
+		"@babel/core": "7.26.10",
+		"@babel/runtime": "7.26.10",
+		"@wordpress/babel-plugin-import-jsx-pragma": "5.22.0",
+		"@wordpress/block-serialization-default-parser": "5.22.0",
+		"autoprefixer": "10.4.20",
+		"postcss": "8.4.47",
+		"postcss-loader": "8.1.1",
+		"sass": "1.64.1",
```

Comment:
To match #43398,
```suggestion
		"sass": "1.87.0",
```

Reply:
Oh good catch! I missed that `sass` was in the Jetpack plugin package file. It is a dev dependency in the forms package and is using the `1.64.1` version. 

https://github.com/Automattic/jetpack/blob/trunk/projects/packages/forms/package.json#L95

I'll update the version in this package to match Jetpack plugin's version.

Reply:
Updated in [0735334](https://github.com/Automattic/jetpack/pull/43413/commits/07353346486079ae3752e3b1af07481526b4629f).

Reply:
Forms confusingly has two entries for sass 😀
https://github.com/Automattic/jetpack/blob/3c9dbc5bc0de562d5321088ef53bfd1667881d63/projects/packages/forms/package.json#L66
https://github.com/Automattic/jetpack/blob/3c9dbc5bc0de562d5321088ef53bfd1667881d63/projects/packages/forms/package.json#L95

Reply:
For a PHP package it doesn't really matter. Since all the JS gets bundled for the released package anyway, technically they're all dev deps.